### DISCUSSION TITLE
How were the developers made aware of the modifications

### DISCUSSION THREAD

**Discussion by sruthipeddinti** [community/community#123594] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When I make changes to a file and another developer makes modifications to the same file, the changes are automatically merged and updated. How were the changes, such as pop-up messages or notifications, known to the developers?

**Comment by ebndev** [comment]
Hey @sruthipeddinti, are you creating pull requests to the repository? If you are watchers of the repo, it should notify you of all issues and PRs created.
